### PR TITLE
Refactor form layouts

### PR DESCRIPTION
**Pull request by mirhamasala**
Refactor forms

Why:

Because we're repeating code throughout the components

*

This change addresses the need by:

* Creating a `FormLayout`

### CODE REVIEW

File: frontend/src/components/Forms/FormLayout/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import Heading2 from "components/Typography/Heading2";
+import Heading3 from "components/Typography/Heading3";
+
+import styles from "./index.module.css";
+
+const FormLayout = ({ children, headerText, promptText }) => {
+  return (
+    <>
+      <div className={styles.headerWrapper}>
+        <Heading2>{headerText}</Heading2>
```
Comment (mirhamasala) [inline frontend/src/components/Forms/FormLayout/index.js:11]: The styles for the `header` and `prompt` probably need to be changed per new design but I'll leave that up to you.

**Review by jreis13 — APPROVED** [review]
I like this, looking cleaner!

File: frontend/src/components/Forms/FormLayout/index.js (hunk 1 above)
Comment (jreis13) [reply frontend/src/components/Forms/FormLayout/index.js:11]: They will but we can tackle that once we have everything up and running on this page + privacy policy page.

**Comment by beasrodrigues24** [comment]
I really like how this looks as well :smile:

**Review by beasrodrigues24 — APPROVED** [review]

**Pull request merged**